src/test/java/io/github/sskorol/testcases/AmazonTests.java

```diff
@@ -33,7 +35,7 @@ public void shouldSearchForParfume(final Parfume parfume, final Account account)
         open(LoginPage.class)
                 .login(account.getUsername(), account.getPassword());
 
-        customAssertThat(account)
+        customAssertThat(SUCCESS)
```

Comment:
you've mixed up expected and actual results... asserts accept actual first, and then verify it against expected...